components/Header.tsx

```diff
@@ -1,17 +1,51 @@
-import { Flex, Heading } from "@chakra-ui/react";
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  Heading,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { useLogout, useUserQuery } from "@lib/hooks/user";
 import { Colors, useColor } from "@styles/colors";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+import Login from "./Login";
 
 export default function Header() {
+  const { isAuthenticated } = useUserQuery();
+  const { isOpen, onClose, onOpen } = useDisclosure();
+
+  const [loginModalKey, setLoginModalKey] = useState("");
+  const handleLoginClick = () => {
+    // Reset login modal each time it is opened
+    setLoginModalKey(uuid());
+    onOpen();
+  };
+
+  const logout = useLogout();
+
   return (
-    <Flex
-      backgroundColor={useColor(Colors.secondaryBackground)}
-      padding={4}
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Heading as="h1" size="md">
-        Book Bazar
-      </Heading>
-    </Flex>
+    <Box as="header" backgroundColor={useColor(Colors.secondaryBackground)}>
+      <Container py={5} maxWidth="container.md">
```

Comment:
Not sure if it's your goal to make it look pretty right now, but I believe the container should be made larger so that there can be more spacing between the
<img width="1456" alt="Screen Shot 2022-01-27 at 12 55 04 PM" src="https://user-images.githubusercontent.com/40670885/151419000-489afaa3-9db2-4254-b823-b69f7d1a2b49.png">
 title and the login button.

Reply:
Oh yeah this header is mostly just a placeholder dw

Reply:
Regardless, I think the container size is good and will look better when there's content on the page but we can revisit this at that point.